### PR TITLE
Add Q.ncall, like nfcall but accepts a thisArg

### PR DESCRIPTION
**Pull request by kylehg**
Hello @nicks, 

Please review the following commits I made in branch 'kyle-call'.

6598adac7a32aeff7cd2532315254f761a60bb5c (2015-08-24 13:32:56 -0700)
Add Q.call, like nfcall but accepts a thisArg

R=@nicks

MANUAL TESTING=not manually tested

### CODE REVIEW

**Comment by nicks** [comment]
hmmm...i think the idiomatic naming convention would be ncall?

see nbind and nfbind
https://github.com/kriskowal/q/wiki/API-Reference#qnfbindnodefunc-args

**Comment by kylehg** [comment]
Ah makes sense, fixed!

File: kew.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ if (typeof process !== 'undefined') {
  * @param {Object|undefined} scope
  * @param {Array} args
  */
-function nextTickCallback (defer, callback, scope, args) {
+function nextTickCallback(defer, callback, scope, args) {
```
Comment (nicks) [inline kew.js]: O.o

**Comment by nicks** [comment]
lg

can you also update the README?

File: kew.js (hunk 1 above)
Comment (kylehg) [inline kew.js]: Sorry for the ocd, I thought our style-guide only puts a space if it's an anonymous function, but now I can't find anything like that...

**Comment by kylehg** [comment]
Added a new release, ready for you to `npm publish` whenever you get a chance.

**Pull request merged**